xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -208,12 +208,18 @@
       #set($discard = $allMatchingParams.addAll($filterParams))
     #end
   #end
+  #if ($request.debug) ##Mode Debug
```

Comment:
Because I didn't find this one (I add this in documentation: http://extensions.xwiki.org/xwiki/bin/view/Extension/Livetable+Macro#HHowtodebuganissuewiththeLivetabledisplay)